tests/components/tomorrowio/test_weather.py

```diff
@@ -67,21 +73,45 @@ def _enable_entity(hass: HomeAssistant, entity_name: str) -> None:
 
 async def _setup(hass: HomeAssistant, config: dict[str, Any]) -> State:
     """Set up entry and return entity state."""
+    data = _get_config_schema(hass, SOURCE_USER)(config)
+    data[CONF_NAME] = DEFAULT_NAME
+    config_entry = MockConfigEntry(
+        domain=DOMAIN,
+        data=data,
+        options={CONF_TIMESTEP: DEFAULT_TIMESTEP},
+        unique_id=_get_unique_id(hass, data),
+        version=1,
+    )
+    config_entry.add_to_hass(hass)
+    assert await hass.config_entries.async_setup(config_entry.entry_id)
+    await hass.async_block_till_done()
+
+
+async def _setup_modern(hass: HomeAssistant, config: dict[str, Any]) -> State:
```

Comment:
Today's modern is tomorrow's legacy. Pun intended. Just checking if HA devs have a sense of humor.

Reply:
Yeah, you're right, this is not a great name